ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.view.js

```diff
@@ -89,7 +89,8 @@ module.exports = Marionette.LayoutView.extend({
         'change:choice',
         function(confirmation) {
             if (confirmation.get('choice')) {
-                this.model.resetToDefaults();
+                const defaults = this.model.get('type') === 'custom' ? this.model.toJSON(): undefined;
```

Comment:
There _might_ be an issue here if you create a new form (which has a type of `new-form`) and then choose reset because the `associatedFormModel` attribute will be cleared [back to undefined](https://github.com/codice/ddf/blob/3a76039db34cca326d9f4b5698a36990b85ac853/ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js#L78) when we had originally passed down the form model when [new form was selected](https://github.com/codice/ddf/blob/3a76039db34cca326d9f4b5698a36990b85ac853/ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js#L60)

Reply:
Verified this isn't an issue ✅ because new forms just have default model values anyway, and we account for that during the save-to-backend [serialization logic](https://github.com/codice/ddf/blob/3a76039db34cca326d9f4b5698a36990b85ac853/ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js#L209) anyway